src/Illuminate/Session/DatabaseSessionHandler.php

```diff
@@ -142,14 +142,14 @@ public function write($sessionId, $data)
      *
      * @param  string  $sessionId
      * @param  string  $payload
-     * @return bool|null
+     * @return int
```

Comment:
Do we want to change the behaviour of this function? What's the use case for this specific one? I think the old interface was better?

Reply:
I find the current behavior of only returning a value in one case a bit odd. I think both cases should return the same type, but that could also be a boolean (as in `DatabaseStore::put()`).